metaflow/metadata_provider/metadata.py

```diff
@@ -672,6 +673,38 @@ def _register_system_metadata(self, run_id, step_name, task_id, attempt):
         if metadata:
             self.register_metadata(run_id, step_name, task_id, metadata)
 
+    @classmethod
+    def filter_tasks_by_metadata(
+        cls,
+        flow_id: str,
```

Comment:
nit - we use `flow_name`

Reply:
Will change it to `flow_name` - I did see `flow_id` in a bunch of different places in `local.py` though.